docs/source/environments/braintumor.rst

```diff
@@ -0,0 +1,163 @@
+
+.. math::
+
+   \begin{aligned}
+   c_t(x, t) &= \overbrace{D\nabla^2 c}^{\text{diffusion}} + \overbrace{\rho c}^{\text{proliferation}} - \overbrace{R(\alpha, \beta, d(x, t), t) c}^{\text{radiation}}\,,  \quad && (x, t) \in \mathcal{B} \times \mathcal{T} \\
+   c(x, 0) &= c_0\,, \quad && x \in \mathcal{B}  \\ 
+   \mathbf{n} \cdot \nabla c(x, t) &=  0\,,  \quad &&  (x, t) \in \partial \mathcal{B} \times \mathcal{T}
+   \end{aligned}
+
+where a zero-flux (Neumann) boundary condition is imposed at the spatial boundaries to ensure that no tumor cells enter or leave the domain during the simulation. :math:`\nabla c` is the spatial gradient of the tumor concentration and :math:`\mathbf{n}` is the outward-pointing unit normal vector at the boundary :math:`\partial \mathcal{B}` of the spatial domain.
+
+Radiation term :math:`R` comes from:
+
+.. math::
+   R(\alpha, \beta,  d(x, t), t) =
+   \begin{cases}
+   1 - S(\alpha, \beta, d(x, t)), & \text{if } t \in \mathcal{T}_{\text{therapy}} \subset \mathcal{T} \\ 
+   0, & \text{otherwise}
+   \end{cases}
+
+.. math::
+  \begin{aligned}
+   S(\alpha, \beta, d(x, t)) = e^{-\alpha \cdot \gamma(\alpha, \beta, d(x, t))} \\
+   \gamma(\alpha, \beta, d(x, t)) = n \cdot d(x,t) \cdot \left( 1 + \frac{d(x, t)}{\alpha / \beta} \right)
+  \end{aligned}
+
+
+**Definitions:**
+
+- :math:`D`: diffusion coefficient in units :math:`(mm^2/year)`
+- :math:`\rho`: proliferation rate in units :math:`(1/year)`
+- :math:`c_0`: initial distribution 
+- :math:`d(x, t)`: fractionated radiation dose in Gray :math:`(Gy = J/kg)` applied at spatial location :math:`x` and time :math:`t`. This value represents the portion of the total prescribed dose delivered at each treatment instance. It reflects the real-world approach of splitting radiation into smaller daily doses rather than applying a full dose at once. Dose distribution varies spatially depending on the tumor's size, and temporally based on treatment schedule
+- :math:`\alpha`: radio-sensitivity parameter in :math:`(Gy^{-1})` that qualifies probability of immediate (single-hit) cell death caused by radiation. In the linear-quadratic model for radiation efficacy, this term governs the linear component of damage
+- :math:`\beta`: radio-sensitivity parameter in :math:`(Gy^{-2})` that qualifies probability of quadratic (double-hit) cell death, where two radiation-induced sub-lethal damages combine to cause cell death. In the linear-quadratic model for radiation efficacy, this term governs the quadratic component of damage
+- The function :math:`\gamma(\cdot)` represents the biologically effective dose (BED) which scales the efficacy of radiation treatment by number of fractions given n, :math:`\alpha`, and :math:`\beta`
+- The function :math:`S(\cdot)` represents the probability a tumor cell survives given that it received radiation dose :math:`d`
+- The function :math:`R(\cdot)` represents the probability a tumor cell dies given that it received radiation dose :math:`d`
+
```

Comment:
you are missing a discssion of non bold n. Also, change this value to a different letter, you are already using bold n for the normal vector.